test/webidl/post.js

```diff
@@ -186,13 +186,13 @@ console.log('int_array[0] == ' + arrayClass.get_int_array(0));
 console.log('int_array[7] == ' + arrayClass.get_int_array(7));
 
 try {
-  arrayClass.set_int_array(-1, struct);
+  arrayClass.set_int_array(-1, 42);
```

Comment:
Without this change this check not reports the bad type error rather than the "index out of range" error which is checked for later on.